files/image_config/monit/memory_checker

```diff
@@ -33,6 +34,15 @@ from swsscommon import swsscommon
 EVENTS_PUBLISHER_SOURCE = "sonic-events-host"
 EVENTS_PUBLISHER_TAG = "mem-threshold-exceeded"
 
+CGROUP_DOCKER_MEMORY_DIR = "/sys/fs/cgroup/memory/docker/"
```

Comment:
Can we add UT for all key functions?

Reply:
done, since there's no build time UT in current repo, after discussion with Qi just verified test code manually and updated result into PR.